### ISSUE TITLE
Keycloak 21.0.1 Class cast exception for class HttpResponse

### ISSUE THREAD

**Issue by cphuoc91** [p2-inc/keycloak-magic-link#22]
There was an error at startup since keycloak-services 21.0.1 have changed Cors class accept classes `org.keycloak.http.HttpRequest` instead of `org.jboss.resteasy.spi.HttpRequest`

**Comment by xgp** [comment]
Yes. There are a few things related to that change that need to be updated in all of our extensions. An update is in process.

**Comment by xgp** [comment]
our `0.8` release solves this, and because of the breaking change in the Keycloak `21.0.0` API, it will no longer be compatible with previous Keycloak versions.
